ripemd160/src/lib.rs

```diff
@@ -20,5 +20,8 @@ pub extern "C" fn main() {
     hasher.input(&data);
     let hash = hasher.result();
 
-    ewasm_api::finish_data(&hash)
+    let mut padded_hash = vec![0; 12];
```

Comment:
Can you please add a comment above this? Explaining that the 20 bytes hash has to be left padded with zeroes to 32 bytes?

Reply:
Sure thing, added.